data_analysis/night_record.py

```diff
@@ -97,7 +97,7 @@ def __init__(
                 continue
             self.min_records.append(m_rec)
 
-        m_lines = docs.get_file(morning_file)
+        docs.get_file(morning_file)
         for line in n_lines:
```

Comment:
Right here it looks like you aren't using the output from the morning file at all and are just running through the night file lines again.

Reply:
Fixed!